prediction_market_agent/agents/metaculus_agent/deploy.py

```
@@ -1,84 +1,83 @@
 from prediction_market_agent_tooling.markets.markets import MarketType
 from prediction_market_agent_tooling.markets.metaculus.metaculus import (
     MetaculusAgentMarket,
 )
-from prediction_market_agent_tooling.tools.utils import check_not_none
 
-from prediction_market_agent.agents.think_thoroughly_agent.think_thoroughly_agent import (
-    ThinkThoroughlyWithItsOwnResearch,
+from prediction_market_agent.agents.prophet_agent.deploy import (
+    DeployablePredictionProphetGPTo1PreviewAgent,
 )
 from prediction_market_agent.utils import DEFAULT_OPENAI_MODEL
```

Ruff

`prediction_market_agent.utils.DEFAULT_OPENAI_MODEL` imported but unused

Remove unused import: `prediction_market_agent.utils.DEFAULT_OPENAI_MODEL`

(F401)